runtime/onert/core/src/exec/ExecutorBase.cc

```diff
@@ -93,7 +94,10 @@ void ExecutorBase::execute(const std::vector<backend::IPortableTensor *> &inputs
     output_tensor->setTensor(output);
   }
 
-  executeImpl();
+  // Create observee
+  ExecutionObservee subject(_observers);
```

Comment:
I'd like to reduce unnecessary overhead from this method. How about moving creating `subject` into constructors of each Executor if that's OK?

Reply:
I didn't add `subject` to `Executor` because `subject` is dependent on each inference. 

I hope `Executor` will not be updated after compilation as possible, and want to remove all dynamic field finally. Then `ExecutionContext` will handle all dynamic feature (including dynamic memory allocation) and it will be created when inference is started and passed to `Executor` as parameter (someday...). Currently, since we start to support dynamic feature such as dynamic shape, `Executor` becomes to be changed after compilation.

Reply:
Thanks for your explanation.To be Honest, I didn't understand the reasons you explained :) 
Anyway, the overhead of creating `ExecutionObservee` may be small, so I think it can be ignored. Sorry for bothering you.